testutil/nullify/nullify.go

```diff
@@ -0,0 +1,57 @@
+// Package nullify provides methods to init nil values structs for test assertion.
+package nullify
+
+import (
+	"reflect"
+	"unsafe"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+var (
+	coinType  = reflect.TypeOf(sdk.Coin{})
+	coinsType = reflect.TypeOf(sdk.Coins{})
+)
+
+// Fill analyze all struct fields and slices with
+// reflection and initialize the nil and empty slices,
+// structs, and pointers.
```

Comment:
[nit] is it possible to improve the grammar here? I'm not sure what this meant to say so can't provide a meaningful suggestion